samples/source-control/external/build.gradle.kts

```diff
@@ -0,0 +1,60 @@
+import org.eclipse.jgit.api.Git
+
+buildscript {
+    repositories { jcenter() }
+    dependencies {
+        classpath("org.eclipse.jgit:org.eclipse.jgit:4.9.0.201710071750-r")
+    }
+}
+
+plugins {
+    java
+}
+
+group = "org.gradle.kotlin.dsl.samples.source-control"
+version = "1.0"
+
+tasks {
+    "generateGitRepo" {
+
+        inputs.dir("src")
+        inputs.files("build.gradle.kts", "settings.gradle.kts")
+
+        val gitRepoDir = file("$buildDir/git-repo")
+        outputs.dir(gitRepoDir)
+
+        doLast {
+            delete(gitRepoDir)
+            delete(temporaryDir)
```

Comment:
:x: Is `delete(temporaryDir)` really necessary?

Reply:
Yes, otherwise jgit throws.